DashboardClient/DashboardClient.cpp

```diff
@@ -145,7 +145,29 @@ namespace Umati
 				auto it = pDataSetStorage->values.find(pNode);
 				if (it == pDataSetStorage->values.end())
 				{
-					// LOG(INFO) << "Couldn't write value for " << pNode->SpecifiedBrowseName.Name << " | " << pNode->SpecifiedTypeNodeId.Uri << ";" << pNode->SpecifiedTypeNodeId.Id;
+					LOG(DEBUG) << "Couldn't write value for " << pNode->SpecifiedBrowseName.Name << " | " << pNode->SpecifiedTypeNodeId.Uri << ";" << pNode->SpecifiedTypeNodeId.Id << "Try to search it with NodeId!";
+					// FIX_BEGIN(FIX_2) In case we don't wnt to remove the duplicate pointers with FIX_1, we can simply check the
+					// Identity of the node via its node Id.
+					//
+					//
+					auto pSimpleNode = std::dynamic_pointer_cast<const ModelOpcUa::SimpleNode>(pNode);
+					if(pSimpleNode) {
+						//std::cout << pSimpleNode->NodeId << "\n";
+						auto values = pDataSetStorage->values;
+						for(auto it1 : values) {
+							auto pSimpleNode1 = std::dynamic_pointer_cast<const ModelOpcUa::SimpleNode>(it1.first);
+								if(pSimpleNode1->NodeId == pSimpleNode->NodeId) {
+									// DEBUG_BEGIN in case we want to see the different pointer addresses.
```

Comment:
Why not use LOG(DEBUG) ?

Reply:
Removed and LOG(DEBUG used). Also removed old classes.